lib/NormalModule.js

```diff
@@ -440,8 +454,17 @@ class NormalModule extends Module {
 				source.insert(end + 0.5, "\n/* WEBPACK VAR INJECTION */" + varEndCode);
 			}
 		}
-		block.blocks.forEach((block) => this.sourceBlock(
-			block, availableVars.concat(vars), dependencyTemplates, source, outputOptions, requestShortener));
+
+		block.blocks.forEach((block) =>
```

Comment:
just making this a little more readable - hard to grok with such high arity if all on one line